submodules/wizard/wizard.js

```diff
@@ -133,72 +133,74 @@ define(function(require) {
 			});
 
 			monster.waterfall([
-				function(waterfallCallback) {
+				function showLoadingTemplate(waterfallCallback) {
```

Comment:
Not sure this task is required in the waterfall chain since the insertion callback is not used (hence making it synchronous).

IMO, we could use the following to still show a loading template, until `navigationWizard.render` is invoked:
```js
monster.ui.insertTemplate($container, null, {
    hasBackground: false
});
```

Reply:
I thought the same, that this task was not necessary :sweat_smile: But then noticed that actually, yes, the purpose is to show a loading template before `navigationWizard.render` is invoked.